### PR TITLE
Add integrations

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* [![Build Status](https://travis-ci.com/rabbitmq/cuttlefish.svg?branch=fix-tests)](https://travis-ci.com/Kyorai/cuttlefish)
```
Comment (dumbbell) [inline README.md]: The first URL stills points to the rabbitmq organization on Travis CI, but it reflects the status of our fork, not upstream.

**Review by dumbbell — CHANGES_REQUESTED** [review]

**Comment by michaelklishin** [comment]
@dumbbell I've corrected the build image URL.

**Comment by michaelklishin** [comment]
I accidentally merged #12 into my local `develop` branch, let me undo that.

**Review by michaelklishin — APPROVED** [review]

**Comment by dumbbell** [comment]
There is still #13 merged in that pull request.

File: rebar.config [hunk 2]
```diff
@@ -5,7 +5,7 @@
             debug_info,
             warn_untyped_record]}.
 
-{deps, [getopt, {lager, "~>3.6.0"}]}.
+{deps, [getopt, {lager, "~>3.7.0"}]}.
```
Comment (dumbbell) [inline rebar.config:8]: Can you please rebase the branch? It has another unrelated branch merged into it.

**Review by dumbbell — CHANGES_REQUESTED** [review]

**Comment by michaelklishin** [comment]
I resubmitted this with one extra commit as #14.

**Pull request closed without merging**